README.md

```diff
@@ -1,5 +1,125 @@
-# mali-error-mapper
-Mali gRPC error mappers and middleware
+## Mali error mapper
+
+Mali gRPC error mappers and middleware.
+
+## Middleware
+
+This middleware maps all non `GrpcError` errors to a `GrpcError` formatted accordingly.
+
+## Status
+
+[![Build Status](https://travis-ci.com/uphold/mali-error-mapper.svg?branch=master)](https://travis-ci.com/uphold/mali-error-mapper)
+
+## Installation
+
+Install the package via `yarn`:
+
+```sh
+❯ yarn add mali-error-mapper
+```
+
+or via `npm`:
+
+```sh
+❯ npm install mali-error-mapper --save
+```
+
+## Mappers
+
+There are three mappers available by default.
+
+- GenericErrorMapper
```

Comment:
Actual implementation seems to be missing.

Reply:
Yes makes sense since this code was removed.